pkg/frontend/installversions_get_test.go

```diff
@@ -0,0 +1,91 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/frontend/adminactions"
+	"github.com/Azure/ARO-RP/pkg/metrics/noop"
+	mock_adminactions "github.com/Azure/ARO-RP/pkg/util/mocks/adminactions"
+)
+
+func TestGetAvailableInstallVersions(t *testing.T) {
+	mockSubID := "00000000-0000-0000-0000-000000000000"
+	mockTenantID := "00000000-0000-0000-0000-000000000000"
+	method := http.MethodGet
+	ctx := context.Background()
+
+	type test struct {
+		name                    string
+		mocks                   func(*test, *mock_adminactions.MockKubeActions)
+		wantStatusCode          int
+		wantResponse            []byte
+		wantResponseContentType string
+		wantError               string
+	}
+
+	for _, tt := range []*test{
+		{
+			name:                    "hard coded versions",
+			mocks:                   func(t *test, mka *mock_adminactions.MockKubeActions) {},
+			wantStatusCode:          http.StatusOK,
+			wantResponseContentType: "application/json",
+			wantResponse:            []byte(`["4.10.15","4.9.28","4.8.18","4.7.30","4.6.44","4.5.39","4.4.33"]`),
```

Comment:
Need to adjust this to only expect the value in `version.InstallStream`, which is currently "4.10.15".

Reply:
Yup, done.